cipher.h

```diff
@@ -59,15 +59,18 @@
 #define CIPHER_DECRYPT		0
 
 typedef struct Cipher Cipher;
-typedef struct CipherContext CipherContext;
+// typedef struct CipherContext CipherContext;
 
 struct Cipher;
+/*
```

Comment:
I think we can probably just nuke these.

Reply:
I agree.

I limited myself to commenting them out because I was not sure.